modular/translations/code/translation_data/ru_names.toml

```
@@ -12039,12 +12326,12 @@ prepositional = "комбинезоне робототехника"
 gender = "male"
 
 ["medical mask"]
-nominative = "медицинская маска"
-genitive = "медицинской маски"
-dative = "медицинской маске"
-accusative = "медицинскую маску"
-instrumental = "медицинской маской"
-prepositional = "медицинской маске"
+nominative = "анестезиологическая маска"
```

она ведь не только для анастезии, можно кислородный баллон подключить. ИМХО, мимо
```suggestion
nominative = "медицинская маска"
```

---

поправил